### PR TITLE
Cgo#71: Indoor Directions Elevator Preferred Settings

### PR DESCRIPTION
**Pull request by DariushN**
# PR Updates
## Application Wide Changes
- Create Shared Preferences Service to store settings on disk
- Created Settings page and added "Prioritize elevators" option
- Reroute all multi floors indoor navigation using escalators or elevators depending on preference
- Created Widget test for Settings menu

### CODE REVIEW

**Review by mteolis — APPROVED** [review]
LGTM

File: lib/widgets/screens/quick_menu.dart [hunk 1]
```diff
@@ -69,6 +69,10 @@ class QuickMenuState extends State<QuickMenu> {
             leading: Icon(Icons.settings),
             title: const Text('Settings'),
             // TODO(Noorzada): add link
```
Comment (danielwiktorczyk) [inline lib/widgets/screens/quick_menu.dart]: Is this TODO still needed?

**Review by danielwiktorczyk — APPROVED** [review]
Looks great to me! Tested scenario Webster to H902 both with and without elevator prioritization, and works as intended!

File: lib/widgets/screens/quick_menu.dart (hunk 1 above)
Comment (DariushN) [reply lib/widgets/screens/quick_menu.dart]: No, good catch! I'll remove it now

**Pull request merged**